spec/spec_helper.rb

```diff
@@ -25,6 +25,13 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  # Randomize specs run order.
+  config.order = :random
+  Kernel.srand config.seed
```

Comment:
I feel this code bit tricky. I guess the intention is to fix seed of Kernel#rand. However IIRC at the moment there is no dependency for such randomness.